src/OrbitGl/TimeGraph.cpp

```diff
@@ -512,7 +515,7 @@ void TimeGraph::UpdatePrimitives(Batcher* /*batcher*/, uint64_t /*min_tick*/, ui
       std::max(capture_max_timestamp_, capture_data_->GetCallstackData().max_time());
 
   time_window_us_ = max_time_us_ - min_time_us_;
-  world_start_x_ = GetPos()[0];
+  world_start_x_ = viewport_->GetWorldTopLeft()[0];
```

Comment:
Just for later reference: There's no need for this member variable to exist any more.